MSAL/src/requests/MSALRequestParameters.m

```diff
@@ -42,6 +44,11 @@ - (void)setScopesFromArray:(NSArray<NSString *> *)scopes
     self.scopes = [[NSOrderedSet alloc] initWithArray:scopesLowercase copyItems:YES];
 }
 
+- (void)setUnvalidatedAuthority:(NSURL *)unvalidatedAuthority
+{
+    _unvalidatedAuthority = unvalidatedAuthority;
```

Comment:
just curious, the property is writable, why do we need to have this setter?

Reply:
Was testing something and forgot to removed, thanks for caching, fixed.